juju/application.py

```diff
@@ -24,6 +25,42 @@
 
 
 class Application(model.ModelEntity):
+    @property
+    def name(self) -> str:
+        return self.safe_data["name"]
+
+    @property
+    def exposed(self) -> bool:
+        return self.safe_data["exposed"]
+
+    @property
+    def owner_tag(self) -> str:
+        warnings.warn("Deprecated", DeprecationWarning)
```

Comment:
Should this be `stacklevel=2` to point to the caller of `owner_tag`? Also, would a more explicit warning message be helpful? "Application.owner_tag is deprecated" (and can we suggest they use something else?)

Reply:
Yes I need to make this better.

Reply:
done